aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -9,7 +9,8 @@ proposals:
     execution_mode: MultiStep
     update_sequence:
       - Gas:
-          new: current
+          old: https://raw.githubusercontent.com/aptos-labs/aptos-networks/main/gas/v1.16.1-rc.json
+          new: https://raw.githubusercontent.com/aptos-labs/aptos-networks/main/gas/v1.18.0.json
       - Framework:
           bytecode_version: 6
           git_hash: ~
```

Comment:
Is the `allow_serialized_script_args` feature flag still needed? I had the impression that it was enabled in the previous release

Reply:
I am not sure - its in the default template.

Reply:
It's from https://github.com/aptos-labs/aptos-core/pull/14097. I think we need to include it. @runtian-zhou to double check.

> I had the impression that it was enabled in the previous release

Guess I had this impression because this feature flag is also on the main branch. We'll need to remember to remove it for 1.19.

Reply:
@runtian-zhou confirmed. We are all green.